cirq-core/cirq/protocols/json_test_data/MeasurementKey.repr

```diff
@@ -0,0 +1 @@
+[cirq.MeasurementKey('key'), cirq.MeasurementKey('nested:key')]
```

Comment:
Was this generated by actually calling `repr(g)` for some object `g` ?

The `_repr_` above returns `f'cirq.MeasurementKey(name={self.name})'`, which seems wrong. See #4440